### PR TITLE
fix: [ANDROAPP-2770] Disable settings parameter when there is no internet connection

### PR DESCRIPTION
**Pull request by Balcan**
[jira issue](https://jira.dhis2.org/browse/ANDROAPP-2770)

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java [hunk 1]
```diff
@@ -115,6 +124,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
     @Override
     public void onResume() {
         super.onResume();
+        context.registerReceiver(networkReceiver, new IntentFilter("android.net.conn.CONNECTIVITY_CHANGE"));
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java]: I think this was deprecated in 28 
https://developer.android.com/reference/android/net/ConnectivityManager#CONNECTIVITY_ACTION

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java]: the value of the constant action is actually Constant Value: "android.net.conn.CONNECTIVITY_CHANGE"

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java]: In fact it is the registration of the receiver in the manifest what is deprecated. It is valid to use if it is registered programmatically using a valid context.

**Comment by ferdyrod** [comment]
Tested!

**Pull request merged**